### PR TITLE
MeterpreterCommandDependencies rubocop rule ran against second batch of modules

### PR DESCRIPTION
**Pull request by cgranleese-r7**
This PR builds upon work related to #15295.

First batch PR: #15479

This is the second batch of modules the new MeterpreterCommandDependencies rubocop rule is being ran against. 

To get these 40 files, I just ran the cop against the local exploits: 
```
rubocop -A --only Lint/MeterpreterCommandDependencies modules/exploits/*/local
``` 

This batch if the first 40 files that the cop amended from with the 107 files the command above made corrections to.
To get those 40 files I used the following command:
```
git diff --staged --name-only | head -n 40 | xargs
``` 

Command to skip modules with railgun api calls.
```
git diff --name-only | xargs grep -L 'railgun' | xargs grep -l "^      'Compat' => {"  | xargs git add
```

Then I just grab the result and passed it into my cop:
```
rubocop -A --only Lint/MeterpreterCommandDependencies modules/exploits/android/local/binder_uaf.rb modules/exploits/android/local/futex_requeue.rb modules/exploits/android/local/janus.rb modules/exploits/android/local/put_user_vroot.rb modules/exploits/linux/local/abrt_raceabrt_priv_esc.rb modules/exploits/linux/local/asan_suid_executable_priv_esc.rb modules/exploits/linux/local/bash_profile_persistence.rb modules/exploits/linux/local/bpf_priv_esc.rb modules/exploits/linux/local/exim4_deliver_message_priv_esc.rb modules/exploits/linux/local/glibc_ld_audit_dso_load_priv_esc.rb modules/exploits/linux/local/glibc_realpath_priv_esc.rb modules/exploits/linux/local/libuser_roothelper_priv_esc.rb modules/exploits/linux/local/nested_namespace_idmap_limit_priv_esc.rb modules/exploits/linux/local/netfilter_priv_esc_ipv4.rb modules/exploits/linux/local/ntfs3g_priv_esc.rb modules/exploits/linux/local/overlayfs_priv_esc.rb modules/exploits/linux/local/ptrace_sudo_token_priv_esc.rb modules/exploits/linux/local/servu_ftp_server_prepareinstallation_priv_esc.rb modules/exploits/linux/local/sudo_baron_samedit.rb modules/exploits/linux/local/udev_netlink.rb modules/exploits/multi/local/xorg_x11_suid_server.rb modules/exploits/osx/local/cfprefsd_race_condition.rb modules/exploits/unix/local/opensmtpd_oob_read_lpe.rb modules/exploits/windows/local/adobe_sandbox_adobecollabsync.rb modules/exploits/windows/local/agnitum_outpost_acs.rb modules/exploits/windows/local/alpc_taskscheduler.rb modules/exploits/windows/local/anyconnect_lpe.rb modules/exploits/windows/local/applocker_bypass.rb modules/exploits/windows/local/bits_ntlm_token_impersonation.rb modules/exploits/windows/local/bthpan.rb modules/exploits/windows/local/bypassuac.rb modules/exploits/windows/local/bypassuac_comhijack.rb modules/exploits/windows/local/bypassuac_dotnet_profiler.rb modules/exploits/windows/local/bypassuac_eventvwr.rb modules/exploits/windows/local/bypassuac_fodhelper.rb modules/exploits/windows/local/bypassuac_injection.rb modules/exploits/windows/local/bypassuac_injection_winsxs.rb modules/exploits/windows/local/bypassuac_sdclt.rb modules/exploits/windows/local/bypassuac_sluihijack.rb modules/exploits/windows/local/bypassuac_windows_store_filesys.rb
```

## Note
As seen in the tests in #15295, there is support to alter/add sections of methods depending on various situations. e.g. 
"The cop verifies if a there is an initialise method, if no initialise present, it should be generated and appended appropriately."

I have tried to identify as many of these scenarios as possible, but if there are anymore edge-cases that may not have been covered in the tests, please call them out and I can get tests added as needed 🕵️ 

## Verification

List the steps needed to make sure this thing works

- [ ] Code review.
- [ ] Verify no api calls have been missed.

### CODE REVIEW

File: modules/exploits/linux/local/bash_profile_persistence.rb [hunk 1]
```diff
@@ -39,6 +39,11 @@ def initialize(info = {})
           'Compat'       =>
           {
             'PayloadType' => 'cmd'
```
Comment (adfoster-r7) [inline modules/exploits/linux/local/bash_profile_persistence.rb:41]: We're missing a `,` here 👀

Comment (cgranleese-r7) [reply modules/exploits/linux/local/bash_profile_persistence.rb]: Got that fixed. Cheers 👍

Comment (dwelch-r7) [reply modules/exploits/linux/local/bash_profile_persistence.rb]: did you fix it manually or was it a problem with the rubocop rules you could fix?

Comment (cgranleese-r7) [reply modules/exploits/linux/local/bash_profile_persistence.rb]: I fixed that one manually, but I'm looking into getting it added into the cop at the moment

Comment (cgranleese-r7) [reply modules/exploits/linux/local/bash_profile_persistence.rb]: Got a fix pushed up to the [rubocop](https://github.com/rapid7/metasploit-framework/pull/15295) rule to handle that scenario 👍

**Comment by adfoster-r7** [comment]
Looks like the rubocop error can be ignored, it's suggesting to update the `sudo_baron_samedit` module with `Stability` information
